packages/sass-variables/_typography.scss

```diff
@@ -263,3 +341,10 @@ $puppet-type-code-small-font-weight: $puppet-type-weight-normal !default;
     pointer-events: none;
   }
 }
+
+@mixin puppet-type-link-scaled() {
+  @include puppet-type-link();
+  font-size: inherit;
+  line-height: inherit;
+  font-weight: inherit;
+}
```

Comment:
I don't think we need this. The `@puppet-type-link` mixin should already work independent of line-height.